src/Button/ToggleButton/ToggleButton.jsx

```diff
@@ -38,15 +34,63 @@ class ToggleButton extends React.Component {
    * @type {Object}
    */
   static propTypes = {
+    className: PropTypes.string,
+    /**
+     * A name to be identified by the togglegroup.
+     * @type {String}
+     */
     name: PropTypes.string,
+    /**
+     * The font awesome icon name.
+     * @type {String}
+     */
     icon: PropTypes.string,
+    /**
+     * The font awesome icon name on pressed state. Use either this or icon.
+     * @type {String}
+     */
     pressedIcon: PropTypes.string,
+    /**
+     * The classname of an icon of an iconFont. Use either this or icon.
+     * @type {String}
+     */
     fontIcon: PropTypes.string,
+    /**
+     * Whether the button should be initially pressed.
+     * @type {bool}
+     */
     pressed: PropTypes.bool,
-    onToggle: PropTypes.func,
+    /**
+     * The onToggle callback.
+     * Be aware that evt can be null if the
+     * Params are: (pressedState {bool}, evt {ClickEvent|null})
+     *
+     * @type {Function}
+     */
+    onToggle: PropTypes.func.isRequired,
+    /**
+     * The tooltip to be shown on hover.
+     * @type {String}
+     */
     tooltip: PropTypes.string,
-    tooltipPlacement: PropTypes.string,
-    className: PropTypes.string
+    /**
+     * The position of the tooltip.
+     * @type {String}
+     */
+    tooltipPlacement: PropTypes.oneOf([
+      'top',
+      'left',
+      'right',
+      'bottom',
+      'topLeft',
+      'topRight',
+      'bottomLeft',
+      'bottomRight',
+      'leftTop',
+      'leftBottom',
+      'rightTop',
+      'rightBottom'
+    ])
```

Comment:
The same as above.. Is there some default value for placement? Should it be required?

Reply:
Default is `top`. See https://ant.design/components/tooltip/